File: CodeForces/1-100/61/A/61A.java
// Problem 61A: Ultra-Fast Mathematician
// https://codeforces.com/contest/61/problem/A
// Compiled with: openjdk 17.0.16 2025-07-15
// Executed with: javac 17.0.16
// Linux version: Linux kernel 6.1.0-10-amd64
// Submitted on: October 22nd, 2025
// Codeforces language used: Java 21 64bit

import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        StringBuilder answer = new StringBuilder("");
        String num1 = sc.nextLine();
        String num2 = sc.nextLine();

        for(int i = 0; i < num1.length(); i++){
            if (num1.charAt(i) == num2.charAt(i)){
                answer.append("0");
            }else{
                answer.append("1");
            }
        }
        System.out.println(answer.toString());
        sc.close();
    }
}

File: CodeForces/1-100/61/A/61A.py
# Problem 61A: Ultra-Fast Mathematician
# https://codeforces.com/contest/61/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 26th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

num1 = str(input())
num2 = str(input())

ans = ""
for index in range(0, len(num1)):
    if num1[index] == num2[index]:
        ans = ans + "0"
    else:
        ans = ans + "1"

print(ans)
